src/app/shared/components/forms/form-body-application/form-body-application.component.html

```diff
@@ -8,25 +8,144 @@
   </div>
   <div class="row">
     <div class="form-group mt-3">
-      <label class="form-label" for="name">{{'QUESTION.GIVE-APPLICATION-NAME' | translate}}</label>*
+      <label class="form-label" for="name">{{'GEN.NAME' | translate}}*</label>
 
       <input type="text" class="form-control" id="name" name="name"
-        [placeholder]="'QUESTION.APPLICATION-NAME-PLACEHOLDER' | translate" maxlength="50" required
+        [placeholder]="'QUESTION.APPLICATION.NAME-PLACEHOLDER' | translate" maxlength="50" required
         [(ngModel)]="application.name"
         [ngClass]="{'is-invalid' : formFailedSubmit && errorFields.includes('name'), 'is-valid' : formFailedSubmit && !errorFields.includes('name')}">
     </div>
 
     <div class="form-group mt-3">
-      <label class="form-label" for="description">{{'QUESTION.GIVE-APPLICATION-DESCRIPTION' | translate}}</label>
+      <label class="form-label" for="description">{{'GEN.DESCRIPTION' | translate}}</label>
       <textarea id="description" name="description"
-        [placeholder]="'QUESTION.APPLICATION-DESCRIPTION-PLACEHOLDER' | translate" [maxLength]="1024" [rows]="6"
+        [placeholder]="'QUESTION.APPLICATION.DESCRIPTION-PLACEHOLDER' | translate" [maxLength]="1024" [rows]="6"
         class="form-control" [(ngModel)]="application.description"
         [ngClass]="{'is-invalid' : formFailedSubmit && errorFields.includes('description'), 'is-valid' : formFailedSubmit && !errorFields.includes('description')}"></textarea>
     </div>
+
+    <div class="form-group mt-3">
+      <label class="form-label" for="application.status">{{'APPLICATION.METADATA-FIELD.STATUS' | translate}}</label>
+      <mat-select class="form-control" name="application.statusOptions" [(ngModel)]="application.status"
+        [ngClass]="{'is-invalid' : formFailedSubmit && errorFields.includes('application.status'), 'is-valid' : formFailedSubmit && !errorFields.includes('application.status')}">
+        <mat-option *ngFor="let statusOption of statuses" [value]="statusOption.value">
+          {{statusOption.label}}
+        </mat-option>
+      </mat-select>
+    </div>
+
+    <div class="form-group mt-3">
```

Comment:
Date is not validated - I can put in non-date string in the text field and it will just not show up in the DB.
Should show error if date cannot be validated